### DISCUSSION TITLE
test_all_types() table function should include new VARINT type.

### DISCUSSION THREAD

**Discussion by rpbouman** [duckdb/duckdb#13310] [Ideas]
The test_all_types built-in is very useful for experimenting with different types and their values.
Now that a new VARINT type is introduced, it should probably include examples of that.

Now it doesn't:

`select * from (describe select * from test_all_types()) where column_type like 'VARINT%';`

```
┌─────────────┬─────────────┬─────────┬─────────┬─────────┬─────────┐
│ column_name │ column_type │  null   │   key   │ default │  extra  │
│   varchar   │   varchar   │ varchar │ varchar │ varchar │ varchar │
├─────────────┴─────────────┴─────────┴─────────┴─────────┴─────────┤
│                              0 rows                               │
└───────────────────────────────────────────────────────────────────┘
```

(Tested on v1.0.1-dev3779 fa831b7681)

**Comment by pdet** [comment]
Absolutely.

@szarnyasg maybe this should be converted to an issue?

**Reply by szarnyasg** [reply]
Yes, thanks for bringing this to my attention!

**Comment by rpbouman** [comment]
Closing discussion as it's already in progress.
